release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -502,6 +502,10 @@ The vSphere Problem Detector Operator periodically checks functionality of {prod
 
 The Local Storage Operator now includes a must-gather image, allowing you to collect custom resources specific to this Operator for diagnostic purposes. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1756096[*BZ#1756096*] for more information.
 
+[id="ocp-4-7-aws-efs-tp_removed-1st"]
+==== External provisioner for AWS EFS (Technology Preview) feature has been removed
+The Amazon Web Services (AWS) Elastic File System (EFS) technology preview feature has been removed and is no longer supported.
```

Comment:
For this and the other PR versions as well:
`s/technology preview feature/Technology Preview feature/` as this is an official RH support status and is generally capitalized.